### ISSUE TITLE
Improved ebuilds

### ISSUE THREAD

**Issue by sjnewbury** [kwohlfahrt/ebuilds#1]
Hi Kai,
Some time ago I borrowed some of your ebuilds for my [gentoo-gpu overlay](https://github.com/sjnewbury/gentoo-gpu).  Since then they've diverged somewhat, perhaps you'd like to merge my changes or combine efforts?

**Comment by kwohlfahrt** [comment]
Yeah, that sounds like a plan. What is the overlap? So far I have:
- opencl-icd
- beignet
- glslang
- SPIRV-Tools
- vulkan-loader

**Comment by sjnewbury** [comment]
On Fri, 2016-10-07 at 05:02 -0700, Kai wrote:

> Yeah, that sounds like a plan. What is the overlap? So far I have:
> opencl-icd
> beignet
> glslang
> SPIRV-Tools
> vulkan-loader
> —
> I have all those, and all multilib where applicable.

dev-libs/..
beignet
libclc
ocl-icd

dev-util/..
SPIRV-Tools
clinfo
glslang
intel-ocl-runtime
nvidia-cuda-sdk
nvidia-cuda-toolkit

media-gfx/..
(a live gimp ebuild - which I should move to gentoo-playground)

media-libs/..
libglvnd
(libmypaint - needed by the live gimp ebuild)
mesa
vulkan-loader

media-video/..
ffmpeg          (for OpenCL support)

sys-devel/..
llvm (need to drop this one since portage version is okay now AFAIK)

x11-drivers/..
nvidia-drivers      (with system glvnd support)

x11-libs/..
libdrm
libva-vdpau-driver  (This needs dropping since I have a version
                in gentoo-playground which I'm attempting to fix
                to work with Chromium/VAAPI)

**Comment by kwohlfahrt** [comment]
Sorry, I meant what the overlap is, so I can have a look at what the differences are between our versions.

**Comment by sjnewbury** [comment]
[gentoo-gpu.diff.txt](https://github.com/kwohlfahrt/ebuilds/files/516496/gentoo-gpu.diff.txt)
